examples/example1/publisher_example.py

```diff
@@ -1,11 +1,13 @@
 import config
+import datetime
 from lab_share_lib.rabbit.schema_registry import SchemaRegistry
 from lab_share_lib.rabbit.basic_publisher import BasicPublisher
 from lab_share_lib.rabbit.avro_encoder import AvroEncoderBinary
 from lab_share_lib.constants import RABBITMQ_HEADER_VALUE_ENCODER_TYPE_BINARY
 from lab_share_lib.types import RabbitServerDetails
 
-MESSAGE = "my message"
+TIMESTAMP = datetime.datetime.now()
```

Comment:
Should this be `utcnow()` to avoid issues with timezones/daylight savings?  It's only an example I know, but it might lead someone to duplicate it for the actual publisher.

Reply:
Ok